stytchauth/stytch-swift
# [feat] User management & send method support

This PR primarily serves to add user-management and send-method support, however there are a number of smaller additions that ladder up to support these additions. The list of changes is below:

## Changes
- [User get and factor deletion methods](https://linear.app/stytch/issue/SDK-649)
- [add send method support](https://linear.app/stytch/issue/SDK-648)
- [delete biometric registration remotely upon local removal](https://linear.app/stytch/issue/SDK-647)
- Add XCTest request testing helpers
- Add LocalStorage type for generic, global cached storage
- Improve AsyncVariants codegen to handle no argument label
- Add generic Identifier type to ensure type safety when deleting factors by id
- Add `activeSessionPresent` SessionStorage helper

## TODO
- [add user management to demo app in near future](https://linear.app/stytch/issue/SDK-651)
